docs/source/workbench/instrumentviewer.rst

```diff
@@ -7,61 +7,58 @@ Instrument Viewer Widget
 Overview
 --------
 
-Mantid visualisation system allows users to view the geometry of an instrument and also select and query individual detectors.
-An instrument is always shown in the context of a workspace containing data recorded with this instrument.
+The Mantid visualisation system allows users to view the geometry of an instrument and also select and query individual detectors.
+An instrument is always shown in the context of a workspace, which contains data recorded with this instrument.
 The detectors are coloured according to the data in the corresponding spectra of the workspace.
 The user can select individual detectors by clicking on them and request to see the data recorded by this detector as a table or a graph.
 The instrument view also allows users to see close-ups of any component.
 
 
 Instrument View Pane
 --------------------
-The instrument view window can be obtained by right clicking on the workspace of interest and selecting "Show instrument".
+The instrument view window can be obtained by right clicking on the workspace of interest and selecting ``Show Instrument``.
 A 2D projection of the detector arrangement will be visible with each detector pixel color coded depending on the integrated number of counts in its corresponding spectrum, as in the image below.
 
 .. figure:: ../images/Workbench/InstrumentViewer/Overview.png
     :align: center
     :width: 635
 
-It is possible to alter the view of the instrument by holding down, either, mouse button and moving (dragging) it within the window.
-There are other mouse and key board controls and these are listed at the bottom of the instrument view window.
-There are other ways to change the view, find and get information about components and these are described in the next sections.
+It is possible to alter the view of the instrument by holding down either mouse button and moving (dragging) it within the window.
+There are other mouse and key board controls, which are listed at the bottom of the instrument view window.
 
-The control panel of the instrument view has four tabs: Render, Pick, Mask, and Instrument Tree.
+The control panel of the instrument view has four tabs: Render, Pick, Mask, and Instrument.
 
 
 Render Tab
 ----------
 The Render tab contains controls for managing the on-screen appearance of the instrument and collected data.
 The top-most combo-box control allows the user to select the way the instrument is projected onto the screen.
-The default setting is "Full 3D" which gives a 3D view of the instrument in an orthogonal projection.
+The default setting is ``Full 3D`` which gives a 3D view of the instrument in an orthogonal projection.
 The other options are "unwrapped views" projecting the instrument onto a curved surface and then unwrapping it onto the screen.
```

Comment:
Should unwrapped views be formatted differently? e.g. ``unwrapped views``

Reply:
Nah, it's just a turn of phrase here, rather than referencing anything on the GUI. That sentence is poor though, so I'll rewrite it anyway

Reply:
I think for turns of phrase we generally use single quotes - certainly makes it less confusing that double quotes which potentially could have been used accidentally instead of the code formatting...